### PR TITLE
First docs

### LINKED ISSUE #2: Add CONTRIBUTING.md
Adding a CONTRIBUTING.md file is very important. This helps all the contributors to the project to understand how to consistently contribute to the project.

### LINKED ISSUE #3: Add MAINTAINERS.md
The maintainers.md file contains the list of the main maintainers of this project.

### PR DESCRIPTION
**Pull request by Sevenarth**
This pull request adds some initial documentation required for the project development.

This PR closes issues #2, #3.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Pushing the changes
+
+When your patch, a collection of interdependent commits, is ready, you may create a pull request to master. Every pull request must be reviewed by another member for approval to merge. If CI testing is employed, a pass is also required for the approval.
+
+### Reverting the changes
+
+If any of your commits in your pending PR have comments that require attention, you must amend your changes to keep a clean history. For example, if your commit `Add calculate_formula function`, meaning that a new function is being added, is lacking a proper documentation, you must amend it instead of creating a new one. This will ensure that your function will come together as a whole into one single commit, instead of being split into multiple commits that are about the same thing resulting in a decline of history consistency.  
+
+nioadfoudisfusdjhfdshgfioudj
```
Comment (Sevenarth) [inline CONTRIBUTING.md]: Oops!

Comment (Sevenarth) [reply CONTRIBUTING.md]: Also the commit name is not fine! Change with something like: Add CONTRIBUTING.md

**Review by Sevenarth — COMMENTED** [review]
Hey, I left some comments on your code

**Comment by Sevenarth** [comment]
Please create an issue for your changes and reference it!

**Comment by Sevenarth** [comment]
This PR closes the issue #2

**Review by amrllkmn — APPROVED** [review]
Looks good!

**Review by roryort — APPROVED** [review]
Great!

**Pull request merged**